### DISCUSSION TITLE
The Twelve (Generative) Days of Christmas - 2024 Edition

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#48] [Comments]
# The Twelve (Generative) Days of Christmas - 2024 Edition

An update to my post showing how generative AI handles the classic Christmas song.

https://www.raymondcamden.com/2024/12/16/the-twelve-generative-days-of-christmas-2024-edition

**Comment by boyzoid** [comment]
It would have been funny AF if two of the images matched perfectly.

**Comment by boyzoid** [comment]
Meta AI can't count.

I see 5 geese and 8 swans.

**Reply by cfjedimaster** [reply]
They _all_ had issues with numbers, not just Meta.
